src/pages/home/report/ReportActionItem.tsx

```diff
@@ -689,6 +689,8 @@ function ReportActionItem({
                     message = 'parentReportAction.reversedTransaction';
                 } else if (ReportActionsUtils.isTrackExpenseAction(parentReportAction)) {
                     message = 'parentReportAction.deletedExpense';
+                } else if (parentReportAction?.childType === CONST.REPORT.TYPE.INVOICE) {
+                    message = 'parentReportAction.deletedInvoice';
```

Comment:
I'm not sure if we need `deleted` updates in this PR, wdyt?

Reply:
Yeah, looks like we cannot delete an invoice currently. I will revert this change.

Reply:
Done - 7b71142, 78f5f64.